cishouseholds/pipeline/survey_responses_version_digital_ETL.py

```diff
@@ -0,0 +1,8 @@
+from cishouseholds.derive import assign_column_uniform_value
+from cishouseholds.pipeline.survey_responses_version_2_ETL import transform_survey_responses_generic
+
+
+def digital_specific_cleaning(df):
```

Comment:
This outer function needs referencing in input_file_stages.py `transformation_functions` to be called on digital input processing